coalib/bearlib/abstractions/Linter.py

```diff
@@ -363,8 +358,7 @@ def process_output_regex(
             Processes the executable's output using a regex.
```

Comment:
why should someone override it?
if someone wants custom parsing behaviour, there's no need for doing that. Use your own `process_output` which allows way more control^^